lib/Frontend/ArgsToFrontendOptionsConverter.cpp

```diff
@@ -410,6 +411,14 @@ void ArgsToFrontendOptionsConverter::computeTBDOptions() {
       Diags.diagnose(SourceLoc(), diag::error_unsupported_option_argument,
                      A->getOption().getPrefixedName(), value);
     }
+  } else if (Args.hasArg(OPT_enable_experimental_cxx_interop) ||
+             Args.hasArg(OPT_cxx_interoperability_mode)) {
+    // TBD validation currently emits erroneous diagnostics when C++ interop is
```

Comment:
I would argue that the diagnostics are **not** erroneous, but rather that IRGen is incorrectly applying linkage and/or visibility to the symbols. I think we need to clarify in the comment that this is specifically a workaround for **correct** diagnostics, and that it also must be a ``FIXME``.

Reply:
Thanks, that makes sense, I updated the comment.